css/styles.css

```diff
@@ -0,0 +1,137 @@
+* {
+  font-family: var(--body-font);
+}
+
+:root {
+  --jumbotron-font: 'Lobster', cursive;
+  --body-font: 'Open Sans', sans-serif;
+}
+
+
+/* Navbar */
+.navbar-toggler {
+  border-radius: 50%;
+  padding: 0;
+}
+
+.nav-link {
+  color: var(--bs-gray-100);
+}
+
+
+/* Jumbotron */
+.jumbotron_font {
+  font-size: 7rem;
+  opacity: 0.8;
+  font-family: var(--jumbotron-font);
+  text-shadow: -2px -1px 2px rgba(0,0,0,0.4);
+}
+
+#jumbotron {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.35), rgba(0, 0, 0, 0.35)), url(../img/vw-bug.jpg);
+}
+
+
+/* Background images */
+#bg_1 {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.35), rgba(0, 0, 0, 0.35)), url(../img/hitchhiker.jpg);
+}
+
+#bg_2 {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.35), rgba(0, 0, 0, 0.35)), url(../img/city-backpack.jpg);
+}
+
+#bg_3 {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.35), rgba(0, 0, 0, 0.35)), url(../img/red-flannel.jpg);
+}
+
+#bg_4 {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.35), rgba(0, 0, 0, 0.35)), url(../img/roadside.jpg);
+}
+
+
+/* Parallax effect */
+.parallax {
+  height: 100vh;
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+
+/* Text box */
+.text_bg {
+  background-color: rgb(0, 0, 0, 0.55);
+  width: 62%;
+  padding: 3rem;
+}
+
+.date {
+  color: var(--bs-gray-500);
+}
+
+.text_main {
+  color: var(--bs-gray-400);
+}
+
+.border {
+  border-color: var(--bs-gray-700) !important;
+  padding: 0.75rem 1.7rem;
+}
```

Comment:
Two things:
- Great job utilizing BS selectors that are provided as suggestions in the documentation
- Using `!important` when necessary, but not too much xD